src/Tools/dotnet-trace/DiagnosticProfiles.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal class DiagnosticProfileProvider
+    {
+        public string Name;
+        public EventLevel Level;
+        public long Keywords;
+    }
+
+    internal class DiagnosticProfile
+    {
+        public DiagnosticProfileProvider[] Providers { get; set; }
+        public string Description { get; set; }
+
+        public static Dictionary<string, DiagnosticProfile> AllProfiles = new Dictionary<string, DiagnosticProfile>()
+        {
+            {
+                "gc-pause",
+                new DiagnosticProfile
+                {
+                    Providers = new DiagnosticProfileProvider[] {
+                        new DiagnosticProfileProvider { Name = "Microsoft-Windows-DotNETRuntime", Level = EventLevel.Informational, Keywords = (long)ClrTraceEventParser.Keywords.GC }
+                    },
+                    Description = "Monitor GC pauses and runtime suspensions"
+                }
+            },
+
+            {
+                "loader-binder",
+                new DiagnosticProfile
+                {
+                    Providers = new DiagnosticProfileProvider[]
+                    {
+                        new DiagnosticProfileProvider { Name = "Microsoft-Windows-DotNETRuntime", Level = EventLevel.Informational, Keywords = ((long)ClrTraceEventParser.Keywords.Binder | (long)ClrTraceEventParser.Keywords.Loader) }
+                    },
+                    Description = "Monitor assembly loader and binder logs"
+                }
+            }
+            // TODO: Add more diagnosic profiles here.
```

Comment:
Nit: TODO implies work is incomplete. Although we could choose to add more in the future, the current functionality works fine as-is.
```suggestion
            // Add more diagnosic profiles here.
```